app/jobs/fax_application_job.rb

```diff
@@ -1,10 +1,10 @@
-class SendApplicationJob < ApplicationJob
-  def perform(snap_application:)
-    @snap_application = snap_application
-
+class FaxApplicationJob < ApplicationJob
+  def perform(snap_application_id:)
```

Comment:
it seems inconsistent to pass in the `id` here and the object in the other job. Thoughts on doing one or the other for both?

Reply:
Yea! I was thinking about making both use IDs, but wanted to decrease surface area in the short term. 

I don't trust delayed jobs serialization/deserialization to gracefully handle if/when we enqueue multiple fax jobs for the same snap application.